docs/styles.scss

```diff
@@ -18,3 +17,101 @@ body {
     // navbar spacing
     padding-top: 53px;
 }
+
+@mixin ignore-demo() {
```

Comment:
these styles were messing up my typography demo, so I made a mixin to ignore them...

Reply:
This for sure would've confused me had I not seen your comment. Could you override the header margins in your typography demo page? 

I think you could just do this in `typography-demo.component.scss`:
```
.ignore-demo {
    margin: 0;
}
```

Then totally wipe out the ignore-demo mixin thing in the main styles.scss file.

Reply:
I like that....